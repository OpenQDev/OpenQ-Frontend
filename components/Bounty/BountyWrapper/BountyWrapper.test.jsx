/**
 * @vi-environment jsdom
 */
import React from 'react';

import BountyWrapper from '.';
import Constants from '../../../test-utils/constant';
import ShallowRenderer from 'react-test-renderer/shallow';

// WARNING If you change the mock data for issues you may need to change some
// of this test's getByText invocations to getAllByText.
describe('BountyWrapper', () => {
  const tokenValues = {
    tokenPrices: { '0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39': 0.67 },
    tokens: { '0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39': 8.040000000000001 },
    total: 8.04,
  };

  const bounty = Constants.bounty;

  it('should render BountyDetails and match DOM Snapshot', () => {
    const shallow = new ShallowRenderer();
    shallow.render(
      <BountyWrapper bounty={bounty} address={bounty.bountyAddress} tokenValues={tokenValues}>
        <div></div>
      </BountyWrapper>
    );
    const tree = shallow.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
