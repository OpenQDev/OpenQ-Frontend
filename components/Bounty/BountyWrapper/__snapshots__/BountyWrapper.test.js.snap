// vi Snapshot v1, https://goo.gl/fbAQLP

exports[`BountyWrapper should render BountyDetails and match DOM Snapshot 1`] = `
<div
  className="flex flex-1 flex-col space-y-8 pt-4 pb-8 w-full max-w-[800px] justify-start"
>
  <div
    className="flex flex-col space-y-2 items-center w-full md:border rounded-sm border-gray-700 text-primary pb-8"
  >
    <h1
      className="flex w-full text-2xl justify-center px-12 py-4 md:bg-[#161b22] md:border-b border-gray-700 rounded-t-sm"
    />
    <div />
  </div>
</div>
`;
